content/numpy/concepts/math-methods/terms/ceil/ceil.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: '.ceil()'
+Description: 'Rounds each element in an array to the nearest greater integer.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'Math'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+In NumPy, the **`.ceil()`** function rounds each element in an array to the nearest integer that is greater than or equal to the element.
+
+This function is applied to real numbers.
+
+## Syntax
+
+```pseudo
+numpy.ceil(input_value, out=None, where=True)
```

Comment:
```suggestion
numpy.ceil(input_value)
```
This syntax is incorrect. The parameters `out` and `where` are not applicable for `numpy.ciel( )` . This parameters are typically used in functions like numpy.unfunc .

Reply:
According to the [numpy docs](https://numpy.org/doc/2.0/reference/generated/numpy.ceil.html) the parameters are optional but applicable. But I understand they might be confusing.